src/Command.php

```diff
@@ -126,6 +126,9 @@ private static function cleanPath(string $path, bool $rtrim = false): string
             $path = \rtrim($path, '/');
         }
 
+        $path = \trim($path, '"');
+        $path = '"' . $path . '"';
```

Comment:
For shell commands, PHP already has a function for that, called `escapeshellarg()`